### DISCUSSION TITLE
/usr/local/bin/apachectl: line 95: lynx: command not found

### DISCUSSION THREAD

**Discussion by amartinwest** [Homebrew/discussions#2960] [Everyday usage] [answered]
Just upgraded brew and I am getting the above.

**Comment by amartinwest** [comment, marked as answer]
brew install lynx 

fixed the problem

**Comment by amartinwest** [comment]
However, then get this

apachectl start
dyld[29535]: Symbol not found: _apr_bucket_alloc_aligned_floor
  Referenced from: /usr/local/Cellar/httpd/2.4.52/bin/httpd
  Expected in: /usr/lib/libaprutil-1.0.dylib
/usr/local/bin/apachectl: line 79: 29535 Abort trap: 6           $HTTPD -k $ARGV

**Reply by carlocab** [reply]
Try
```
brew reinstall $(brew deps httpd) httpd
```

**Reply by amartinwest** [reply]
THanks, that worked, came up with a permissions problem on initially.

**Comment by fuzemobi** [comment]
I appreciate the answers above.  However, will there be a homebrew update to address this issue?

**Reply by jasonhemann** [reply]
Looks like no. https://github.com/Homebrew/homebrew-core/issues/170930#issuecomment-2254355488
